packages/query-core/src/__tests__/hydration.test.tsx

```diff
@@ -1220,12 +1213,6 @@ describe('dehydration and rehydration', () => {
       expect(clientQueryClient.getQueryData(query.queryKey)).toBe(0),
     )
 
-    expect(serializeDataMock).toHaveBeenCalledTimes(1)
-    expect(serializeDataMock).toHaveBeenCalledWith(0)
-
-    expect(deserializeDataMock).toHaveBeenCalledTimes(1)
-    expect(deserializeDataMock).toHaveBeenCalledWith(0)
-
```

Comment:
Removed these as the test isn't about whether if serialization is about and the amount of times the mock gets called doubled so it gets a bit weird and unintuitive anyway